ResourceManager/CommitDataRenders/CommitDataBodyRenderer.cs

```diff
@@ -38,7 +38,7 @@ public string Render(CommitData commitData, bool showRevisionsAsLinks)
                 throw new ArgumentNullException(nameof(commitData));
             }
 
-            string body = WebUtility.HtmlEncode((commitData.Body ?? "").Trim());
+            string body = WebUtility.HtmlEncode((GitRevision.FormatBodyAndNotes(commitData.Body, commitData.Notes) ?? "").Trim());
```

Comment:
`GitRevision` is being moved to another library, and this would introduce a cross-library dependency, which we can't have.

Reply:
So where do we move this commonly needed method? I don`t know these plans. What will be the greatest-common-denominator csproj here?

Reply:
GCD seems to be "ResourceManager", but it has nothing to do with resources.
Moved out of future library to GitUI.UIExtensions for now and inlined usages. Not perfect, not technically an extension method.